examples/next/docs/react/accessibility/src/hooksCallbacks.ts

```diff
@@ -0,0 +1,116 @@
+import Handsontable from "handsontable";
+import { SELECTED_CLASS, ODD_ROW_CLASS } from "./constants";
+
+const headerAlignments = new Map([
+  ["9", "htCenter"],
+  ["10", "htRight"],
+  ["12", "htCenter"],
+]);
+
+type AddClassesToRows = (
+  TD: HTMLTableCellElement,
+  row: number,
+  column: number,
+  prop: number | string,
+  value: any,
+  cellProperties: Handsontable.CellProperties
+) => void;
+
+export const addClassesToRows: AddClassesToRows = (
+  TD,
+  row,
+  column,
+  prop,
+  value,
```

Comment:
I cannot build the demo cause I've got linter complains (to strict rules?).
```
src/hooksCallbacks.ts:23:3 - error TS6133: 'prop' is declared but its value is never read.
23   prop,
     ~~~~
src/hooksCallbacks.ts:24:3 - error TS6133: 'value' is declared but its value is never read.
24   value,
     ~~~~~
```

Reply:
will fix, cheers